test/e2e/misc_test.go

```diff
@@ -50,12 +52,41 @@ func (suite *MiscTestSuite) SetupTest() {
 func (suite *MiscTestSuite) TestValidateBuildImage() {
 	buildImage := os.Getenv("BUILD_IMAGE")
 	require.NotEmptyf(t, buildImage, "BUILD_IMAGE must be defined")
-	imagesMap, err := getJaegerOperatorImages(fw.KubeClient, namespace)
-	require.NoError(t, err)
 
-	require.Len(t, imagesMap, 1, "Expected 1 deployed operator")
+	// If we did the normal test setup the operator will be in the current namespace.  If not we need to iterate
+	// over all namespaces to find it, being sure to match on WATCH_NAMESPACE
+	didSetup := len(noSetup) == 0
+	if didSetup {
+		imagesMap, err := getJaegerOperatorImages(fw.KubeClient, namespace)
```

Comment:
Looks fine for now, but it would perhaps be cleaner to have each branch in its own func, like, `assertCaseA()` vs. `assertCaseB()`